lib/react_on_rails/version_checker.rb

```diff
@@ -57,7 +57,7 @@ def initialize(package_json)
       end
 
       def raw
-        JSON.parse(package_json_contents)["dependencies"]["react-on-rails"]
+        JSON.parse(package_json_contents)["dependencies"]["react-relay-on-rails"]
```

Comment:
should we maybe still keep the name to be react-on-rails or make a separate library for react-on-rails-relay?